templates/contrib/opening_hours/opening-hours-widget.html.twig

```diff
@@ -0,0 +1,80 @@
+{#
+/**
+ * @file
+ * Template for the Opening Hours : OpenNow formatter.
+ *
+ * Available variables:
+ * - preview_widget : Contains the preview widget.
+ * - widgets : The widgets to show.
+ * - service_id : The opening hours Service ID.
+ * - channel_id : The opening hours Channel ID.
+ * - date : The date to show the widget for.
+ *
+ * @see template_preprocess_vg_social_link()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{{ attach_library('gent_base/opening_hours') }}
+
+<div class="openinghours-wrapper">
+  {% if display_title %}
+    <h3 class="openinghours-channel-title" data-service="{{ service_id }}" data-channel="{{ channel_id }}"></h3>
+  {% endif %}
+
+  {% if preview_widget %}
+    <div
+        class="openinghours-widget"
+        data-service="{{ service_id }}"
+        {% if not single_widget %}data-channel="{{ channel_id }}"{% endif %}
```

Comment:
Same question as above for channel_id, but I assume this is a property of preview_widget?

Reply:
See [my comment above](https://github.com/StadGent/drupal_theme_gent-base/pull/523/files/aa0e8bba2057e59c3df8beb7369d9c0e7c7dd1f9#r243271613)